README.md

```diff
@@ -79,3 +79,4 @@
 This project follows the [MIT License](/LICENSE).
 
 ## SHOW SOME LOVE BY LEAVING A ⭐
```

Comment:
Noo the readme file of folder in which you're adding code. It's the repo readme. Don't do it here.

Reply:
Remove the link from main repo.

Reply:
Thank you for guiding me! I have made the required changes.Please check it out!